### PR TITLE
Feature/flag sets

### PR DESCRIPTION
**Pull request by chillaq**
# Python SDK

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,13 @@
+9.6.1 (Nov 3, 2023)
```
Comment (mmelograno) [inline CHANGES.txt:1]: 9.6.0

**Review by mmelograno — CHANGES_REQUESTED** [review]

File: splitio/client/config.py [hunk 2]
```diff
@@ -1,14 +1,15 @@
 """Default settings for the Split.IO SDK Python client."""
 import os.path
 import logging
+import re
```
Comment (mmelograno) [inline splitio/client/config.py:4]: this import is not needed

File: CHANGES.txt (hunk 1 above)
Comment (chillaq) [reply CHANGES.txt]: fixed

File: splitio/storage/__init__.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 """Base storage interfaces."""
 import abc
-
+import threading
```
Comment (mmelograno) [inline splitio/storage/__init__.py:3]: unused

File: splitio/util/storage_helper.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    sets_to_fetch = []
+    for flag_set in flag_sets:
+        if not flag_set_filter.set_exist(flag_set) and flag_set_filter.should_filter:
+            _LOGGER.warning("Flag set %s is not part of the configured flag set list, ignoring the request." % (flag_set))
+            continue
+        sets_to_fetch.append(flag_set)
+
+    return sets_to_fetch
+
+def combine_valid_flag_sets(result_sets):
```
Comment (mmelograno) [inline splitio/util/storage_helper.py:57]: same as def combine_valid_flag_sets(result_sets):
duplicated code

**Review by mmelograno — CHANGES_REQUESTED** [review]

**Review by mmelograno — APPROVED** [review]

File: splitio/storage/__init__.py (hunk 3 above)
Comment (chillaq) [reply splitio/storage/__init__.py]: fixed

File: splitio/util/storage_helper.py (hunk 4 above)
Comment (chillaq) [reply splitio/util/storage_helper.py:57]: fixed

File: splitio/client/config.py (hunk 2 above)
Comment (chillaq) [reply splitio/client/config.py]: fixed

**Pull request merged**